.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: Package and Publish to Visual Studio Marketplace
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Set Git configuration
+        run: |
+            git config user.name "${GITHUB_ACTOR}"
+            git config user.email "${GITHUB_ACTOR}@bots.github.com"
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Bump version and create Git tag
+        uses: Aqovia/npm-version-increment@v1.0.4
```

Comment:
Can we not hard code the version? Is there a way to use latest version?

Reply:
All the examples I've seen reference a specific version. I think I tried not specifying the version and it just failed, but I can try to find out how to just use latest.

Reply:
You can specify a version (partial or full), or you can specify a branch and it will use latest for that branch. This has been set to the @tidyup branch for now, as the PR for the master branch of the action has not been completed. Once it has, we can change this to @master.